setup.py

```
@@ -108,16 +140,65 @@ def write_version_files() -> tuple[Path, Path]:
 
     return version_txt_path, version_py_path
 
 
+def get_hs_connectors_requirement() -> str:
+    build_type = os.getenv("SPECULATORS_BUILD_TYPE", "dev").lower()
+    version_py_path = REPO_ROOT / "src" / "speculators" / "version.py"
+
+    if building_from_sdist() and version_py_path.exists():
+        version, tag, build_iteration = read_existing_version(version_py_path)
+    else:
+        version, tag, build_iteration = get_next_version(
```

Ruff (0.16.1)

[warning] Unpacked variable `tag` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)